src/LodeRunner.API/src/Handlers/RequestLogger/RequestLogger.cs

```diff
@@ -209,10 +213,33 @@ private static string GetClientIp(HttpContext context, out string xff)
             // Note: clientIp will be null when running Test Project
             // remove IP6 local address
             return clientIp?.Replace("::ffff:", string.Empty);
+        }
+
+        /// <summary>
+        /// Extracts and returns request body
+        /// </summary>
+        /// <param name="request">HttpRequest</param>
+        /// <returns>Request body as object.</returns>
+        private static object LogRequestPayload(HttpRequest request)
+        {
+            if (request.Method == "POST" || request.Method == "PUT")
+            {
+                // Allows using several time the stream in ASP.Net Core
+                request.EnableBuffering();
+
+                StreamReader reader = new (request.Body);

```

Comment:
Utilize using() statement for streamReader. e.g 
 using (StreamReader sr = new StreamReader(your source))